docs/pages/distribution/optimizing-updates.md

```diff
@@ -0,0 +1,77 @@
+---
+title: Optimizing Updates
+---
+
+## Table of contents
+
+- [What's in an Update?](#whats-in-an-update)
+  - [Estimating the Size of an Update](#estimating-the-size-of-an-update)
+- [Limits on Expo's Updates Service](#limits-on-expos-updates-service)
+  - [Optimize Images](#optimize-images)
+  - [Reduce Large Dependencies](#reduce-large-dependencies)
+  - [Download Files when Needed](#download-files-when-needed)
+  - [Building Your App on Your Own Computer](#building-your-app-on-your-own-computer)
+  - [Self-hosting Your Updates](#self-hosting-your-updates)
+- [A Glimpse at the Future](#a-glimpse-at-the-future)
+
+## What's in an Update?
+
+An update for an Expo app comprises the JavaScript, manifest, images, and other assets that a compatible Expo client app can download and run.
+
+On Android and iOS, the standalone apps you submit to the app stores are examples of client apps that run your updates. On the web, the web browser is the client app that runs updates. In fact, on the web, Expo updates are just web applications!
+
+Updates are embedded in standalone apps and delivered over the internet. Because downloading an update over the internet uses bandwidth and takes time, one of the most effective ways to optimize an update is to make it smaller. This guide covers some ways to reduce the sizes of your updates.
```

Comment:
```suggestion
Updates are embedded in standalone apps and subsequent updates are delivered over the internet after a standalone app has been built. Because downloading an update over the internet uses bandwidth and takes time, one of the most effective ways to optimize update delivery is to make it smaller. This guide covers some ways to reduce the sizes of your updates.
```